distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -2057,11 +2117,11 @@ type Table
        row of the specified column. If `term` is empty, the function returns the
        table unchanged.
 
-       This method follows the exact replacement semantics of the
-       `Text.replace` method.
+       This method follows the exact replacement semantics of `Text.replace`.
 
        Arguments:
-       - columns: The column(s) to replace values on.
+       - columns: Specifies columns by a single instance or Vector of names,
+         indexes or regular expressions to match names.
```

Comment:
```suggestion
       - columns: Specifies columns by a name, index or regular expression to match names, or a vector of these.
```

Very subjective, but IMO the `instance` in this context could be unclear. I had to read it twice before understanding. So maybe it could be worth to reword it slightly.